File: programmers/cupang/1.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> solution(int n, int min_postion, int max_position, vector<int> positions);
int main(void)
{
    ;
    return 0;
}

vector<int> solution(int n, int min_postion, int max_position, vector<int> positions)
{
    int length = max_position - min_postion;
    int gap = length / (n - 1);
    vector<int> houses;
    vector<int> answer;
    vector<bool> check;
    for (int i = min_postion; i <= max_position; i += gap)
        houses.push_back(i);

    check.assign(n, false);

    for (int i = 0; i < positions.size(); i++)
    {
        for (int j = 0; j < n; j++)
        {
            ;
        }
    }
}

File: 2019/programmers/2019bro/1.py
def solution(restaurant, riders, k):
    answer = 0
    for r in riders:
        x, y = r[0], r[1]
        if (x - restaurant[0]) ** 2 + (y - restaurant[1]) ** 2 <= k ** 2:
            answer += 1
    return answer
